core/src/main/resources/error/error-classes.json

```diff
@@ -3,9 +3,29 @@
     "message" : [ "Field name %s is ambiguous and has %s matching fields in the struct." ],
     "sqlState" : "42000"
   },
+  "CANNOT_GET_JDBC_TYPE" : {
+    "message" : [ "Can't get JDBC type for %s" ],
+    "sqlState" : "42000"
+  },
+  "CANNOT_READ_FOOTER_FOR_FILE" : {
+    "message" : [ "Could not read footer for file: %s" ],
+    "sqlState" : "42000"
+  },
+  "CANNOT_TRANSLATE_NON_NULL_VALUE_FOR_FIELD" : {
+    "message" : [ "Can't translate non-null value for field %s" ],
+    "sqlState" : "0A000"
+  },
   "CONCURRENT_QUERY_ERROR" : {
     "message" : [ "Another instance of this query was just started by a concurrent session." ]
   },
+  "DATA_TYPE_UNSUPPORTED_YET" : {
+    "message" : [ "%s is not supported yet." ],
+    "sqlState" : "0A000"
+  },
+  "DDL_UNSUPPORTED_TEMPORARILY" : {
```

Comment:
Same here, but remove "temporarily." We should rename the functions as well to avoid confusion.

Reply:
Thanks for your reviewing. Should I remove "temporarily" in the function's name, as well as in its message? And should I rename that function to "UNSUPPORTED_DDL"? @karenfeng